nais/dev-gcp.json

```diff
@@ -1,28 +1,34 @@
 {
   "app": "pleiepengesoknad-api",
-  "cluster": "dev-sbs",
-  "namespace": "default",
+  "cluster": "dev-gcp",
+  "namespace": "dusseldorf",
   "team": "dusseldorf",
+  "tenant": "trygdeetaten.no",
   "minReplicas": "1",
   "maxReplicas": "2",
   "ingresses": [
-    "https://pleiepengesoknad-api-q.nav.no",
     "https://pleiepengesoknad-api.dev.nav.no",
-    "https://pleiepengesoknad-api.nais.oera-q.local"
+    "https://pleiepengesoknad-api-gcp.dev.nav.no"
+  ],
+  "externalHosts": [
+    "api-gw-q1.oera.no",
```

Comment:
Trenger ikke lenger apigw. 
Kan du ikke kalle direkte?

Reply:
Jo, men lurte på om jeg skulle gjøre det i to omganger slik at det ikke ble alt for mye en gang.

Reply:
Neida, ta hele i samme slengen :)

Reply:
Okei